MegaDeskWebApp/MegaDeskWebApp/Pages/MyDesk/Create.cshtml

```diff
@@ -56,7 +56,7 @@
                     <option value="Rosewood">Rosewood</option>
                     <option value="Veneer">Veneer</option>
                 </select>
-                <div id="centerMaterial"><img id="materialImage" src="" /></div>
+                <div id="centerMaterial"><img id="materialImage" src="https://via.placeholder.com/195x53.png?text=Material" /></div>
```

Comment:
Placeholder image for material. replaced by the selected material
https://via.placeholder.com/195x53.png?text=Material